opam-state.opam

```diff
@@ -28,6 +28,7 @@ build: [
 depends: [
   "ocaml" {>= "4.08.0"}
   "opam-repository" {= version}
+  "opam-solver" {= version}
```

Comment:
That's unfortunate. Is there no way of moving things around to avoid this? I feel like this will cause problem with many plugins that just want opam-state and not all the dependencies and C requirements pulled by opam-solver

Reply:
At current state, we need to compute dependency cone at state stage.